core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -2368,7 +2368,7 @@ private[scheduler] class DAGSchedulerEventProcessLoop(dagScheduler: DAGScheduler
     case ExecutorLost(execId, reason) =>
       val workerHost = reason match {
         case ExecutorProcessLost(_, workerHost, _) => workerHost
-        case ExecutorDecommission(workerHost) => workerHost
+        case ExecutorDecommission(_, host) => host
```

Comment:
See comment below for `ExecutorDecommission` ... Should this be changed to a:

```
case decom @ ExecutorDecommission => decom.workerHost // or decom.host
```

You don't need to add an extra '_' then.

Reply:
Yeah, this actually a good point! This's actually a rule of  Databricks' scala style guide. But I just follow the style of above `ExecutorProcessLost` here. I think it's acceptable when there're not many arugumenetes are being expaned.